### PR TITLE
Aweksey the Femboy medic

### PR DESCRIPTION
**Pull request by ZahaSanko**
## About The Pull Request
Adds Aweksey the Femboy medic to the game in a form of a plushie.

## Why It's Good For The Game
Funny and people like it

## Changelog
:cl:
add: a new plushie
/:cl:
awd

### CODE REVIEW

**Comment by Captiajoe** [comment]
"mUh ImmERsiOn", in all seriousness we tease the poor guy quite a bit already, no need to add jet fuel to this dumpster fire that is his existence. (however that is a cute plushy and good job)

**Comment by chrapacz2000** [comment]
WHERE THE HELL IS CHIPRAPS PLUSHIE!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by Alexeivichi** [comment]
wife approves of this

**Comment by ChungusGamer666** [comment]
honestly quite incredible

**Review by Kuro020 — CHANGES_REQUESTED** [review]
lock it behind fun mode, that's where memes go

**Comment by neohenry** [comment]
> 
> 
> lock it behind fun mode, that's where memes go

lock it inside a jar

**Comment by jroinc** [comment]
This doesn't need to be PR'd, we already *have* him ingame.

File: code/game/objects/items/toys/toys.dm [hunk 1]
```diff
@@ -520,6 +520,12 @@
 	item_state = "rounyplush"
 	attack_verb = list("slashes", "bites", "pounces")
 
+/obj/item/toy/plush/aweksey
+	name = "aweksey the femboy medic"
```
Comment (XSlayer300) [inline code/game/objects/items/toys/toys.dm:524]:
```suggestion
	name = "Aweksey the Femboy Medic"
```

File: code/game/objects/items/toys/toys.dm [hunk 2]
```diff
@@ -520,6 +520,12 @@
...
+	desc = "A soft and breedable Plushie of TGMC's famous femboi."
```
Comment (XSlayer300) [inline code/game/objects/items/toys/toys.dm:525]:
```suggestion
	desc = "A soft and breedable plushie of TGMC's famous femboi."
```
>breedable 
🤔

**Comment by SpaceLoveSs13** [comment]
No

**Comment by Walarks** [comment]
I'll put it in rare xeno research

**Comment by TiviPlus** [comment]
https://www.youtube.com/watch?v=OiKxFJAt7Yg

**Comment by TiviPlus** [comment]
No player references

**Comment by Alexeivichi** [comment]
YES! I'M FREE!

**Pull request closed without merging**